dependencies.gradle

```diff
@@ -13,10 +13,11 @@ ext.dep = [
   "gcpKms": "com.google.apis:google-api-services-cloudkms:v1-rev30-1.23.0",
   "gcpLogback": "com.google.cloud:google-cloud-logging-logback:0.39.0-alpha",
   "gradleBuildScan": "com.gradle:build-scan-plugin:2.3",
-  "grpcGenJava": "io.grpc:protoc-gen-grpc-java:1.14.0",
-  "grpcNetty": "io.grpc:grpc-netty:1.14.0",
-  "grpcProtobuf": "io.grpc:grpc-protobuf:1.14.0",
-  "grpcStub": "io.grpc:grpc-stub:1.14.0",
+  "grpcGenJava": "io.grpc:protoc-gen-grpc-java:1.21.0",
```

Comment:
and you're also updating the version?

Reply:
this is the version we end up using anyway, via one of our transitive deps